Problem3/tc_array_duplicates.rb

```diff
@@ -0,0 +1,48 @@
+# File:  tc_array_duplicates.rb
+
+require_relative "array_duplicates.rb"
+require "test/unit"
+
+class TestArrayDuplicates < Test::Unit::TestCase
+	
+	def test_empty
+		
+		array_duplicates = ArrayDuplicates.new
```

Comment:
Again the naming seems of to me. When I read "array duplicates", in my mind that means maybe a data structure that contains the duplicates found in an array and in now way a class which can determine if there are any duplicates in an array. In my opinion, a better name for this class would be `DuplicateFinder`